ui-components/src/page_components/listing/ListingCard.tsx

```diff
@@ -47,7 +52,15 @@ const ListingCard = (props: ListingCardProps) => {
           </h4>
         )}
         <div className="listings-row_table">
-          {tableProps.data && <GroupedTable {...tableProps} />}
+          {(tableProps.data || tableProps.stackedData) && (
+            <>
+              {tableHeaderProps?.stackedTable ? (
+                <StackedTable {...(tableProps as StackedTableProps)} />
+              ) : (
+                <StandardTable {...(tableProps as StandardTableProps)} />
```

Comment:
I think we'll still want to use `GroupedTable` unless the stacked mode is requested, otherwise we'll lose the special display of unit categories (reserved for seniors, etc.) like so:

<img width="1028" alt="image" src="https://user-images.githubusercontent.com/658496/136324621-3366b687-abab-480b-bae9-abae83294055.png">

Reply:
I believe we purposefully removed this from the UI a while ago @slowbot is this the case?

Reply:
@jaredcwhite Jesse and I connected offline, the orange bar for reserved is indeed deprecated :) We were still using the GroupedTable but we had stopped passing in group units a while ago. The reasoning is that we will only ever use the Reserved indication as a tag above the table and not within the table.